source/captiveportal/captive-portal-configuration.rst

```diff
@@ -0,0 +1,171 @@
+Configuring a Captive Portal Zone
+=================================
+
+When creating a captive portal zone, multiple settings can be edited,
+depending on your needs : 
+
+
+Avaliable timeouts
+------------------
+
+pfSense's captive portal has 3 integrated timeout options : 
+
+- **"Idle timeout"** causes users to be automatically disconnected after an amount of time of network inactivity.
+  The "last activity time of each user" is retrived using the last byte sent by each device. 
+  (meaning last byte "uploaded" from the user perspective)
+- **"Hard timeout"** causes users to be automatically disconnected after a certain amount of time, regardless of their activity.
+- **"Traffic quota"** causes users to be automatically disconnected after exceeding an amount of data (inclusive both download and upload).
+
+
+If one of these options is enabled, pfSense will automatically disconnect users every minute.
+The way pfSense work is that a scheduled task (cron) will run every minute, checking for users to 
+be disconnected.
+
+.. tip:: Each of these timeout are performing user disconnection, but does not prevent them to re-login.
+ In order to perform such thing, please setup a RADIUS server (see below) that will prevent users from re-login.
+
+
+Authentication methods
+----------------------
+
+Four authentication methods are avaliable on pfSense : 
+
+- Don't authenticate users : All users will be redirected on the captive portal
+  login page but will get connected as soon as they click "next". This is useful for
```

Comment:
If the button is labeled "next" this should be formatted as bold, so `**Next**`. If it's named something else like Login then the name should reflect that as well.